.eslintrc

```diff
@@ -15,17 +15,27 @@
     "jsx": true
   },
   "rules": {
-    "quotes": 0,
     "no-underscore-dangle": 0,
     "curly": [2, "multi-line"],
     "camelcase": [2, {"properties": "never"}],
-    "space-in-parens": [2, "never"],
+    "space-in-parens": [2, "always"],
```

Comment:
I think we're putting this in to appease short breaks, who use this in their current linting rules. Nobody over at the wave was so concerned about it that they 100% wanted it kept out.